lib/src/main/ktx/com/fabernovel/statefullayout/StatefulLayout.kt

```diff
@@ -106,3 +109,65 @@ fun StatefulLayout.showLoading(): State = showState(R.id.stateLoading)
  * @return the content state
  */
 fun StatefulLayout.showContent(): State = showState(R.id.stateContent)
+
+/**
+ * Sets a minimal time to wait after each successful state change triggered by [requestStateChange].
+ *
+ * All requests triggered from [requestStateChange] will be handled in respect of this minimal
+ * time, sometimes dropping some requests when many (backpressure).
+ *
+ * Default value for [timeToWaitInMillis] is [DEFAULT_REGULATOR_PACE]
+ *
+ * @see [StateChangeRegulator] for details about regulation rules.
+ */
+@FlowPreview
+@ExperimentalCoroutinesApi
+fun StatefulLayout.setMinimalTimeToWaitBetweenStateChanges(timeToWaitInMillis: Long) {
+    val currentRegulator = stateChangeRegulator
+    if (currentRegulator == null) {
+        val regulator = StateChangeRegulator(timeToWaitInMillis)
+        viewCoroutineScope.launch {
+            regulator.handleStateChangeRequests { request -> processStateChangeRequest(request) }
+        }
+        stateChangeRegulator = regulator
```

Comment:
This should be handled by the `stateChangulator` extension which should getOrPut the regulator inside the view tag.